server/data/ciagApi/interfaces/ciagPlan.ts

```diff
@@ -12,8 +12,10 @@ import WorkInterestsValue from '../../../enums/workInterestsValue'
 import ReasonToNotGetWorkValue from '../../../enums/reasonToNotGetWorkValue'
 
 export default interface CiagPlan {
+  reference?: string
```

Comment:
Most of the changes in this PR revolve around the need to pass `reference` fields for the various resources to our API.